content/abstracts.rst

```diff
@@ -458,3 +474,30 @@ Monday is the Sprint day, we split into small groups and in each group
 a member of an Open Source Python project guides a small group in how
 to hack on the project. Improve your Python skills in a fun, practical
 and effective way.
+
+.. _commitsprint:
+
+Don't be afraid to commit
+-------------------------
+
+A workshop/tutorial for Python/Django developers who would like to
+contribute more to the projects they use, but need more grounding in
+some of the tools required.
+
+The workshop will take participants through the complete cycle of
+identifying a simple issue in a Django or Python project, writing a
+patch with documentation, and submitting it.
+
+Read more here: http://dont-be-afraid-to-commit.readthedocs.org
+
+.. _sciencesprint:
+
+Python for Science Sprint
+-------------------------
+
+Collaborative open science sprint where you can bring along a task to
+automate, or a piece of code you want to open source, and we will put
+you in groups to turn your existing scripts or workflow into an
```

Comment:
Don't think that "we will put you in groups" is the right phrasing.  How about "you will find people to help you"?